lib/doorkeeper/config.rb

```diff
@@ -189,6 +189,8 @@ def extended(base)
     option :force_ssl_in_redirect_uri,      default: !Rails.env.development?
     option :grant_flows,                    default: %w(authorization_code client_credentials)
     option :access_token_generator,         default: "Doorkeeper::OAuth::Helpers::UniqueToken"
+    option :refresh_token_revoked_on_use,   default: false
+    option :refresh_token_revoked_in,       default: 0
```

Comment:
Having `refresh_token_revoked: 0` s equivalent to `refresh_token_revoked_on_use: false`. I would remove the `refresh_token_revoked_on_use` option, and infer it from the revoked in option.